paper/paper.md

```diff
@@ -51,8 +51,8 @@ authors:
     orcid: 0000-0000-0000-0000
     affiliation: 
   - name: Njagi Mwaniki
-    orcid: 0000-0000-0000-0000
-    affiliation: 
+    orcid: 0000-0002-4858-2375
+    affiliation: Pwani University
```

Comment:
Please add your affiliation in lines 68 and following. Here, you can just add the number of the affiliation. Thanks!

Reply:
@urbanslug

Reply:
Done in https://github.com/graph-genome/graph-genome.github.io/pull/24/commits/1d3c1e0787b17527fd41089a6d13192ecb33a63c